mbuild/formats/charmm_writer.py

```diff
@@ -1212,14 +1203,22 @@ def __init__(
             its original order, as in the Compound sent to the writer.
             If True, the order of the atoms is reordered based on their
             residue names in the 'residues' list that was entered.
-        box_0 : list, [x-dim, y-dim ,z-dim]
-            A list of 3 positive float values or the dimensions [x, y ,z]
-            for structure_box_0 in nanometers (nm)
-            This is to add/override or change the structures dimensions. Ex: [1,2,3]
-        box_1 : list, [x-dim, y-dim ,z-dim]
-            A list of 3 positive float values or the dimensions [x, y ,z]
-            for structure_box_1 in nanometers (nm)
-            This is to add/override or change the structures dimensions. Ex: [1,2,3]
+        box_0 : Box
+            The Box class that contains the attributes Lx, Ly, Lz for the length
+            of the box 1 in nm. It also contains the xy, xz, and yz Tilt factors
+            needed to displace an orthogonal box's xy face to its
+            parallelepiped structure for box 0.
+        box_1 : Box
+            The Box class that contains the attributes Lx, Ly, Lz for the length
+            of the box 1 (units in Angstroms (Ang)). It also contains the xy, xz, and yz Tilt factors
+            needed to displace an orthogonal box's xy face to its
+            parallelepiped structure for box 0.
+        box_0_vectors : numpy.ndarray, [[float, float, float], [float, float, float], [float, float, float]]
+            Three (3) sets vectors for box 0 each with 3 float values, which represent
+            the vectors for the Charmm-style systems (units in Angstroms (Ang))
+        box_1_vectors : numpy.ndarray, [[float, float, float], [float, float, float], [float, float, float]]
```

Comment:
Just to make sure, for your boxes, do you need them to be in row or column major format?

mBuild Boxes treat everything as row-major
```
a_x, a_y, a_z
b_x, b_y, b_z
c_x, c_y, c_z
```
If you need it to be column major just take the transpose of the box vectors

```python3
col_major_box = row_major_box.T
```

Reply:
I'm actually extracting the data like this for each  numpy.ndarray value:
 box_0_vectors[0][0]      box_0_vectors[0][1]      box_0_vectors[0][2]
 box_0_vectors[1][0]      box_0_vectors[1][1]       box_0_vectors[1][2]
 box_0_vectors[2][0]      box_0_vectors[2][1]       box_0_vectors[2][2]


I'm not sure I understand, is this not in the row column format?

Reply:
No that is correct. Just wanted to make sure you wanted it in row major.

Reply:
this was just a comment not suggestion, so marking it resolved.